state/controller_test.go

```diff
@@ -29,29 +29,31 @@ func (s *ControllerSuite) TestControllerAndModelConfigInitialisation(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 
 	optional := set.NewStrings(
-		controller.IdentityURL,
-		controller.IdentityPublicKey,
+		controller.AgentRateLimitMax,
```

Comment:
I assume this is just Sort() the list alphabetically

Reply:
Yeah, I hit my OCD limit